imgui-SFML.cpp

```diff
@@ -988,7 +991,7 @@ void Image(const sf::Sprite& sprite, const sf::Vector2f& size, const sf::Color&
 
     const sf::Texture& texture = *texturePtr;
     sf::Vector2f textureSize = static_cast<sf::Vector2f>(texture.getSize());
-    const sf::IntRect& textureRect = sprite.getTextureRect();
+    sf::FloatRect textureRect = static_cast<sf::FloatRect>(sprite.getTextureRect());
```

Comment:
```suggestion
    const sf::FloatRect textureRect = sf::FloatRect(sprite.getTextureRect());
```

There's already a constructor for converting between different types of `Rect`s

Reply:
The same can be said for the variable above.
I did consider
```suggestion
    const sf::FloatRect textureRect{sprite.getTextureRect()};
```
but went for consistency regardless.
Should I apply your suggestions?

Reply:
Yes. It's a lot nicer to use the constructor that was added explicitly for this purpose. Let's not repeat mistakes for the sake of consistency.